community/office-hours.md

```diff
@@ -0,0 +1,30 @@
+# Kubernetes Community Office Hours
+
+Office Hours is a live stream where we answer live questions about Kubernetes from users on the [YouTube channel](https://www.youtube.com/c/KubernetesCommunity/).
+
+### When and Where
+
+Third Wednesday of every month, there are two sessions: 
+
+- European Edition: [1pm UTC](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Kubernetes+Office+Hours+%28EU+Edition%29&iso=20171018T13) 
+- Western Edition: [8pm UTC](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Kubernetes+Office+Hours+%28Western+Edition%29&iso=20171018T13&p1=234)) 
+
+Tune into the [Kubernetes YouTube Channel](https://www.youtube.com/c/KubernetesCommunity/live) to follow along. 
+You can post questions on the [#office-hours channel](https://kubernetes.slack.com/messages/office-hours) on Slack.
+
+### Archives
+
+Archives of all the sessions are kept here:
+
+- [Office Hours playlist](https://www.youtube.com/watch?v=D0Q7wwljN30&list=PL69nYSiGNLP3azFUvYJjGn45YbF6C-uIg)
+- [Office Hours meeting notes](http://bit.ly/k8s-office-hours-notes)
+
+
+### Volunteering
+
+We're always looking for volunteers to host and answer questions. Volunteers looking to host hours in 
+more time zones are also welcome to join in and help us expand. 
+
+- [Volunteer working sheet](http://bit.ly/k8s-office-hours-volunteers)
+
+If you have any questions ping @castrojo.
```

Comment:
Github doesn't linkify markdown docs with usernames. You might want to do something like:
```
**[@castrojo](https://github.com/castrojo)**
```